components/Editor/index.tsx

```diff
@@ -117,17 +117,18 @@ const Editor = ({ readOnly = false }: Props) => {
   )
 
   const getCallValue = useCallback(() => {
-    const _callValue = new BN(callValue)
+    const _callValue = BigInt(callValue)
+    let cv = BigInt.asIntN(32, BigInt(0))
 
     if (unit === ValueUnit.Gwei) {
-      _callValue.imul(new BN('1000000000'))
+      cv = BigInt.asIntN(32, _callValue * BigInt('1000000000'))
```

Comment:
What's the reason for this manipulation?
Why do you clamp result to 32 bits?

Reply:
The reason for removing uses of `BN` in favor of `BigInt` in that file is because of the migration to the new version of `@ethereumjs` which dropped support for `BN`.
As for the clamping, totally by mistake, should have been 256 actually (I probably misread bits for bytes 🤦‍♂️ ) will fix and commit, thanks!